src/index.html

```diff
@@ -5,9 +5,268 @@
+          <div class="person">
+            <img
+              src="images/people/1.jpg"
+              alt=""
+              class="person__photo"
+            >
+            <h3 class="person__name">Tim Cook</h3>
+            <p class="person__position">CEO</p>
+            <p class="person__text">
+              Nice work, Certificates National Association of Realtors (USA).
+            </p>
+          </div>
+          <div class="person">
+            <img
+            src="images/people/2.jpg"
+            alt=""
+            class="person__photo"
+          >
+            <h3 class="person__name">Katherine Adams</h3>
+            <p class="person__position">Senior Vice President</p>
+            <p class="person__text">My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart. I’m all alone and blissfully happy in the local edge.</p>
+          </div>
+          <div class="person">
+            <img
+              class="person__photo"
+              src="./images/people/3.jpg"
+              alt=""
+            >
+            <h3 class="person__name">Jonathan Ive</h3>
+            <p class="person__position">Chief Design Officer</p>
+            <p class="person__text">When my lovely valley of steam rises and half-day sun is above an impermeable.</p>
+          </div>
+        </div>
+      </section>
+      <section class="section">
+        <h2 class="heading section__heading" id="contact">Contact Us<span class="heading__crown"></span></h2>
+        <div class="contact">
+          <div class="contact__text">
+            <p>
+              <a href="mailto:Hello@miami.com" class="link" >Hello@miami.com</a><br>
+              <a href="tel:+1-234-555-55-55" class="contact__number">
+                +1 234 555-55-55
+              </a>
+            </p>
+            <p>400 first ave. n.<br>suite 700<br>Minneapolis, MN 55401</p>
+          </div>
+          <div>
+            <form
+              action="#"
+              method="POST"
+              class="form contact__form"
+            >
+              <label for="contact-name" class="form__item">
+                <input
+                  type="text"
+                  id="contact-name"
+                  name="name"
+                  placeholder="Name"
+                  class="input form__input"
+                >
+              </label>
+              <label for="contact-email" class="form__item">
+                <input
+                type="email"
+                id="contact-email"
+                name="email"
+                placeholder="Email"
+                class="input form__input"
+              >
+              </label>
+              <label for="message">
+                <textarea
+                  class="input input--textarea form__input"
+                  name="messege"
+                  id="message"
+                  placeholder="Messege"
+                >
+                </textarea>
```

Comment:
```suggestion
                <textarea class="input input--textarea form__input" name="messege" id="message" placeholder="Messege"></textarea>
```
с литером плохо, но без него хуже)

Reply:
> с литером плохо, но без него хуже)

так не сохраняет мою работу через git commit, пишет исправь ошибку, а потом попробуй заново сохранить. (как вариант могу сохранить работу как правильно по lint, а исправить например здесь в гитхабе, чтобы высвечивалось в демо правильно)